src/strings/csv/en.csv

```diff
@@ -240,7 +240,7 @@ COMPLIANCE,Compliance,
 COMPROMISED_SEEDS,Compromised Seeds,
 CONDITIONAL,Conditional,
 CONFIRM_IN_FLIGHT_REQUEST,"There is a request in flight, are you sure you want to navigate away from this page?",
-CONFIRMED_RESTORABLE_LAND,Confirmed Restorable Land (ha),
+CONFIRMED_RESTORABLE_LAND,TF Restorable Land (ha),
```

Comment:
Leaving the string name the same for now since it is more closely tied to the name of the back end field. If we end up changing that perhaps we should update the string name too.